tests/test_utils.py

```diff
@@ -0,0 +1,73 @@
+"""Unit tests for the `utils` module."""
+
+import warnings
+
+import datasets.utils.logging as ds_logging
+import transformers.utils.logging as hf_logging
+from datasets.utils import enable_progress_bar
+
+from coral_models.utils import block_terminal_output, ignore_transformers_output
+
+
+class blocking_output:
```

Comment:
snake_case?

There is probably some good reasoning behind this, so just curious :)

Reply:
It seems as though context managers specifically are usually written in snake case, regardless of their implementation being function- or class based. Examples include `warnings.catch_warnings` and `open` (when opening files). Some examples can be found in [PEP 343](https://peps.python.org/pep-0343/#examples).

Reply:
Cool, thanks for the link.

Should we rename it to `blocked_output` or `output_blocked`?
From your link to PEP 343:

> The tense used in the names of the example contexts is not arbitrary. Past tense (“-ed”) is used when the name refers to an action which is done in the __enter__ method and undone in the __exit__ method. Progressive tense (“-ing”) is used when the name refers to an action which is to be done in the __exit__ method.

Reply:
@lfwa Ah, good catch! Renamed it to `output_blocked`.